README.md

```diff
@@ -1,4 +1,4 @@
-# Project 8: Around The U.S.
+# Project 9: Around The U.S.
 
```

Comment:
According to the task you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.